### PR TITLE
Allow for the user to define workflows

### PR DESCRIPTION
**Pull request by cooktheryan**
specifically of workflows within the operator.

Part of #85 

Signed-off-by: Ryan Cook <rcook@redhat.com>

### CODE REVIEW

File: roles/job_runner/tasks/main.yml [hunk 1]
```diff
@@ -7,6 +7,20 @@
...
+    - lookup('env','TOWER_JOB_TEMPLATE_NAME') is defined
+    - lookup('env','TOWER_JOB_TEMPLATE_NAME') != ''
+
+- name: Read AnsibleWorkflow Specs
+  k8s_info:
+    kind: AnsibleWorkflow
+    api_version: tower.ansible.com/v1alpha1
+    name: "{{ lookup('env', 'ANSIBLEWORKFLOW_NAME') }}"
+    namespace: "{{ lookup('env', 'ANSIBLEWORKFLOW_NAMESPACE') }}"
+  register: ansible_workflow
```
Comment (rooftopcellist) [inline roles/job_runner/tasks/main.yml:20]: Is this used anywhere?  Do we need this task?

Comment (rooftopcellist) [inline roles/job_runner/tasks/main.yml:20]: I don't think this is currently referenced anywhere.  @cooktheryan I think you need to use it here: https://github.com/ansible/awx-resource-operator/pull/87/files#diff-72886cf7b8009753cc2039b0b997ee8b841ef4fefffe6f959e52b13f428779b9R6

File: config/rbac/role.yaml [hunk 2]
```diff
@@ -90,6 +90,23 @@ rules:
       - patch
       - update
       - watch
+  ##
+  ## Rules for tower.ansible.com/v1alpha1, Kind: AnsibleWorkflow
+  ##
+  - apiGroups:
+      - tower.ansible.com
+    resources:
```
Comment (rooftopcellist) [inline config/rbac/role.yaml:98]: We can combine these 3 resources with the resources section above (they have the same rules as far as I can tell).

Comment (cooktheryan) [reply config/rbac/role.yaml:98]: resolved in #94

**Pull request closed without merging**